tutos/dev/manual/application.wiki

```diff
@@ -813,6 +814,28 @@ lines are drawn on both windows.
 
 <<concepts| Widgets with Ocsigen-widgets >>
 
+For the rest of the tutorial, we use the following import :
+
+<<code language="ocaml" class="server"|
+[%%shared
+  (* Modules opened in the shared-section are available in client-
+     and server-code *)
+  open Eliom_content.Html.D
+  open Eliom_content
+  open Lwt
```

Comment:
I think this one is not opened in graffiti.
Never open Lwt, and avoid as much as possible opening modules globally.
It makes codes more difficult to read (especially for a tutorial, it is important that the use knows straightaway were the functions come from).